src/main/java/ru/vsu/csd/datatransferservice/service/TextAnalysisService.java

```diff
@@ -0,0 +1,66 @@
+package ru.vsu.csd.datatransferservice.service;
+
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TextAnalysisService {
+    public TextAnalysisService() {
+    }
+
+    public ArrayList<Double> startAnalysis(String text) {
+        System.out.println("Input text: " + text);
```

Comment:
use slfj (read about it) instead of just sout. Add dependency and configure bean for logging